website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -37,9 +37,15 @@ cd $HOME/projects/memcached-operator
 # so all API groups will be <group>.example.com
 operator-sdk init --domain example.com --repo github.com/example/memcached-operator
 ```
+API groups are are a mechanism to group portions of the Kubernetes API. You're probably already familiar with
+some of the core Kubernetes API groups, such as `apps` or `rbac.authorization.k8s.io`. You should name your
+domain to group your resource types in meaningful group(s), both for ease of understanding and because these
+groups determine how access can be controlled to your resource types using RBAC. For more information, see [the core Kubernetes docs](https://kubernetes.io/docs/reference/using-api/#api-groups).
```

Comment:
The RBAC is generated via the markers we have a part/section on the tutorial that only speaks about it. Ditto so, should not any extra info about that fits better there? 

Also, I am not sure if I can go along with `domain to group your resource types in meaningful group(s), both for ease of understanding and because these
groups determine how access can be controlled to your resource types using RBA`

- It will create the endpoints for the k8s API
- The RBAC give permissions to HTTP protocol permissions ( PATCH/DELETE/CRETE ) to these endpoints 
- In the project scaffold by default, we have RBAC permissions that are generated to allow the Operator/manager to perform these operations ( verbs ) for what is required
But we also scaffold permissions to help cluster admins create groups to allow view/edit each GKV scaffold.

Reply:
I am closing this one and I hope that I could clarify all points here: https://github.com/operator-framework/operator-sdk/pull/5842#discussion_r930702102